main.css

```diff
@@ -0,0 +1,29 @@
+#main{
+    text-align: center;
+    background: #eee;
+}
+#img-div{
+    background: white;
```

Comment:
Please use hexadecimal for naming colors instead of color names e.g use `background: #fffff;` instead of `background: white;`. The reason to use hexadecimal numbers is it’s a human-friendly representation of values in binary code. Please check for reference: https://www.pluralsight.com/blog/tutorials/understanding-hexadecimal-colors-simple